### PR TITLE
Add pod name to error messages when getting logs of a container

### PR DESCRIPTION
**Pull request by feihujiang**
Add pod name to error messages when getting logs of a container.

### CODE REVIEW

**Comment by feihujiang** [comment]
@k8s-bot ok to test

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -2091,13 +2091,13 @@ func (kl *Kubelet) GetKubeletContainerLogs(podFullName, containerName, tail stri
 
 	if err := kl.validatePodPhase(&podStatus); err != nil {
 		// No log is available if pod is not in a "known" phase (e.g. Unknown).
-		return err
+		return fmt.Errorf("Pods %q: %v", podFullName, err)
```
Comment (wojtek-t) [inline pkg/kubelet/kubelet.go]: Can you please change: "Pods" to "Pod"?
Same below.

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
@wojtek-t  OK,  changed based on your feedback,  please review again.

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
Changed podFullName to name and namespace (example,  podFullName: test_default,  name:test, namespace:default )

**Comment by wojtek-t** [comment]
LGTM

**Comment by gmarek** [comment]
@k8s-bot test this please

**Pull request merged**